assimilation_code/programs/run_tests.csh

```diff
@@ -79,12 +81,22 @@ if ( ! $?host) then
    setenv host `uname -n`
 endif
 
+echo
+echo
+echo "=================================================================="
+echo "Starting tests of DART programs at "`date`
+echo "=================================================================="
+echo
```

Comment:
I moved these so they are the first things to print.
Some 'build and run logs' statements were appearing before this ... ditto for the 'running on host' messages.